### PR TITLE
front: add stdcm test e2e

### LINKED ISSUE #5342: front: refacto E2E tests: stdcm page
# Description

Refactor operationnal studies, stdcm page, pathfinding e2e tests according to what's been defined in #5022

# Goal

- [ ] add apiContext in each test to make them parallelisable
- [ ] add missing tests fore each user action

# Acceptance criteria

The following e2e have been refectored :
- [ ] stdcm page

### PR DESCRIPTION
**Pull request by clarani**
closes #5342

### CODE REVIEW

File: front/tests/pages/rolling-stock-selector-page.ts [hunk 1]
```diff
@@ -0,0 +1,81 @@
+import { Locator, Page, expect } from '@playwright/test';
+import rollingstockTranslation from '../../public/locales/fr/rollingstock.json';
+import BasePage from './base-page';
+
+const electricCheckboxTranslation = rollingstockTranslation.electric;
+
+export default class RollingStockSelectorPage extends BasePage {
```
Comment (Uriel-Sautron) [inline front/tests/pages/rolling-stock-selector-page.ts:7]: Very good idea the "extends"

**Review by Uriel-Sautron — APPROVED** [review]
Lgtm ! Thanks for your work !!

**Review by Yohh — APPROVED** [review]
lgtm

**Review by Math-R — APPROVED** [review]
Lgtm, nice basepage :)

**Review by Math-R — APPROVED** [review]

**Pull request merged**